include/alpaka/test/mem/view/ViewTest.hpp

```diff
@@ -131,11 +130,7 @@ namespace alpaka::test
                 {
                     if(pBytes[i] != byte)
                     {
-                        printf(
-                            "Byte at offset %lu is different: %u != %u\n",
-                            static_cast<long unsigned>(i),
-                            static_cast<unsigned>(pBytes[i]),
-                            static_cast<unsigned>(byte));
+                        printf("Byte at offset %zu is different: %hhu != %hhu\n", i, pBytes[i], byte);
```

Comment:
`zu` is unfortunately not supported by nvcc :/

Reply:
How is it compiling and running then? Does `nvcc` silently replace this with something else?

Reply:
It didn't the other day with my nvcc 12.x at work. It said that the format specifier `z` is not supported. So I changed it to the `long unsigned` here.

Mind that if the unit tests run successfully, this code is never printed. So successful completion of the test suite does not tell you whether this change is correct.

Reply:
I quickly tried to add a test case for that but it's not so easy to capture the printf output. Also some refactoring of the test suite is necessary. I postpone that effort.